### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by ofek** [ionelmc/pytest-benchmark#119]
@ionelmc Could we please get a new version up on PyPI? :shipit:

**Comment by ionelmc** [comment]
It's on my todolist

**Comment by hamin** [comment]
Last release was over a year ago. Would highly appreciate this :)

**Comment by kwohlfahrt** [comment]
Now that pytest-4.0 is out it would be very nice to have a compatible release. Is there anything I can do to help?

**Comment by ofek** [comment]
Oh no, does this not work on pytest 4?

**Comment by ionelmc** [comment]
I'll check this week to see what's wrong, maybe a release is possible but realistically speaking I can only clear the huge pile of issues during xmas-newyear period.

**Comment by kwohlfahrt** [comment]
I think it was fixed in #129 - the `node.get_marker` stuff, just not yet released. It is possible to work around it by setting:

```
[pytest]
filterwarnings =
    ignore::pytest.RemovedInPytest4Warning
```

according to the pytest docs, but that will only work until 4.1 so we've just pinned `pytest < 4.0` for now. Thanks for the quick response and good to hear you're working on it.

**Comment by ionelmc** [comment]
So 3.2.0 was released. There will be a 3.2.1 today for some things that I forgot.
